### DISCUSSION TITLE
UX Meeting 2 Notes

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/ux-research#13] [General]
This is a place for the team to put their notes.

**Comment by mrmetaverse** [comment]
Shadow made a [figjam](https://www.figma.com/file/L36As6RxgQ0VxfdOuLprgC/OMI-UXResearch_November112022?node-id=0%3A1) for looking through the suggestions made in #12 
![image](https://user-images.githubusercontent.com/63426722/199287785-3af7fd7f-c154-412d-8157-3bb7ab757846.png)

**Reply by mrmetaverse** [reply]
https://www.figma.com/file/L36As6RxgQ0VxfdOuLprgC/OMI-UXResearch_November112022?node-id=0%3A1

**Comment by mrmetaverse** [comment]
I closed out the starter milestone and tickets. The team can decide to use any system they want.

**Comment by mrmetaverse** [comment]
Shadow made a spatial space 
https://www.spatial.io/s/OMI-Meeting-Room-636137fa095ff10001262c07?share=860100311509375879

**Comment by lyuma** [comment]
We added many of the cards into the [figma board](https://www.figma.com/file/L36As6RxgQ0VxfdOuLprgC/OMI-UXResearch_November112022?node-id=0%3A1):

![result image of figma](https://github.com/omigroup/ux-research/discussions/13#discussioncomment-4029094)
